### DISCUSSION TITLE
User agent for Vercel bots

### DISCUSSION THREAD

**Discussion by MaybeThisIsRu** [vercel/vercel#5095] [Ideas] [answered]
I have an issue with the analytics service I'm using. Details here: https://github.com/plausible/analytics/issues/280

1. Is this Vercel/a bot by Vercel?
2. If so, what would the `user-agent` be?

**Comment by TooTallNate** [comment, marked as answer]
It's likely the screenshot taking service for the deployment overview page on the Vercel dashboard.

**Reply by styfle** [reply]
Yes, this is likely the screenshot service.

We don't specify the user-agent, but I made Story 9731 internally for this feature request, thanks!

**Reply by MaybeThisIsRu** [reply]
Thank you @TooTallNate and @styfle.

Where should I expect an update once this is released? As I would need to notify Plausible analytics to take it up further.

**Reply by TooTallNate** [reply]
We will reply here when we update the User-Agent for the screenshot service.

**Reply by TooTallNate** [reply]
@hirusi Hey again, so the screenshot service is now sending the `User-Agent` header. You should see a value of something like `Vercelbot (+https://vercel.com)` which you can use to filter out of your metrics. Let us know if that solves your issue please!

**Reply by MaybeThisIsRu** [reply]
The letters "bot" by themselves in the User Agent was enough for my case. Thank you!
